packages/core/src/components/tag-input/resizableInput.tsx

```diff
@@ -0,0 +1,39 @@
+/* !
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ */
+
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+
+import { Classes, HTMLInputProps } from "../../common";
+
+type ResizableInputProps = HTMLInputProps;
+export type Ref = HTMLInputElement;
```

Comment:
I don't think this typedef needs to exist. Just inline the type: `forwardRef<HTMLInputElement, ResizableInputProps>`